exec/java-exec/src/test/java/org/apache/drill/exec/store/parquet/TestParquetMetadataCache.java

```diff
@@ -495,14 +495,14 @@ public void testFutureUnsupportedMetadataVersion() throws Exception {
     try {
       test("use dfs_test.tmp");
       test("create table `%s` as select * from cp.`tpch/nation.parquet`", unsupportedMetadataVersion);
-      String lastVersion = Iterables.getLast(MetadataVersion.Constants.SUPPORTED_VERSIONS);
-      for (String supportedVersion : MetadataVersion.Constants.SUPPORTED_VERSIONS) {
-        if (new MetadataVersion(lastVersion).compareTo(new MetadataVersion(supportedVersion)) < 0) {
+      MetadataVersion lastVersion = Iterables.getLast(MetadataVersion.Constants.SUPPORTED_VERSIONS);
```

Comment:
Added `ImmutableSortedSet` in the last commit. It allows do not worry about order of elements.